src/components/Label/__stories__/label.stories.mdx

```diff
@@ -77,13 +77,10 @@ A label indicates the status of an item.
 </Canvas>
 
 ### Clickable
-By applying the onClick prop, the label will become clickable and get all the relevant states (hover, focus, ...) and keyboard support
 <Canvas>
   <Story name="Clickable" parameters={{ chromatic: { pauseAnimationAtEnd: true }}}>
-    <div className="monday-storybook-label_group"><Label text="New" onClick={NOOP} /><Label text="New" kind={Label.kinds.LINE} onClick={NOOP} /></div>
-    <div className="monday-storybook-label_group"><Label text="New" color={Label.colors.NEGATIVE} onClick={NOOP} /><Label text="New" color={Label.colors.NEGATIVE} kind={Label.kinds.LINE} onClick={NOOP} /></div>
-    <div className="monday-storybook-label_group"><Label text="New" color={Label.colors.POSITIVE} onClick={NOOP} /><Label text="New" color={Label.colors.POSITIVE} kind={Label.kinds.LINE} onClick={NOOP} /></div>
-    <div className="monday-storybook-label_group"><Label text="New" color={Label.colors.DARK} onClick={NOOP} /><Label text="New" color={Label.colors.DARK} kind={Label.kinds.LINE} onClick={NOOP} /></div>
+    <div className="monday-storybook-label_group monday-storybook-label_states-gap"><Label text="New" onClick={NOOP} /></div>
+    <div className="monday-storybook-label_group monday-storybook-label_states-gap"><Label text="New" kind={Label.kinds.LINE} onClick={NOOP} /></div>
```

Comment:
Designers proposed to remove this?

Reply:
Yes because it's repetitive and we did the same pattern in `Chips`